src/lib/list/selection-list.spec.ts

```diff
@@ -233,6 +233,30 @@ describe('MatSelectionList without forms', () => {
       expect(manager.activeItemIndex).toEqual(1);
     });
 
+    it('should focus and toggle the next item when pressing SHIFT + UP_ARROW', () => {
+      const manager = selectionList.componentInstance._keyManager;
+      const upKeyEvent = createKeyboardEvent('keydown', UP_ARROW);
+      Object.defineProperty(upKeyEvent, 'shiftKey', {get: () => true});
```

Comment:
Would it make sense to add a utility function like `addModifierKey`?

Reply:
I suppose, but I think that we should do a larger wrapper around these events that allows you do more stuff.